task-management/src/main/java/org/exoplatform/task/management/templates/detail.gtmpl

```diff
@@ -54,11 +63,10 @@
         </ul>
         <a class="actionIcon close-right-panel" href="#"><i class="uiIconClose"></i></a>
     </div>
-
-    <div class="uiEditableInline clearfix ">
-        <span class="iconTaskName pull-left"><i title="&{tooltip.clickToEdit}" data-toggle="tooltip" data-placement="bottom" class="uiIconFolder"></i></span>
-        <div class="breadcrumbCont uiEditableInline" data-original-title="&{tooltip.clickToEdit}" data-toggle="tooltip" data-placement="bottom">
-            <ul class=" ${editableClasses} small"
+    <div class="${projectInlineEditClass} clearfix ">
+        <span class="iconTaskName pull-left"><i title="&{tooltip.clickToEdit}" data-toggle="${projectDataToggle}" data-placement="bottom" class="uiIconFolder"></i></span>
```

Comment:
The tooltip on the icon (title) still says that we can click on it to change the project, even if I have not the permission to do it.